openc3/lib/openc3/microservices/router_microservice.rb

```diff
@@ -25,16 +25,15 @@
 module OpenC3
   class RouterMicroservice < InterfaceMicroservice
     def handle_packet(packet)
-      @count += 1
       RouterStatusModel.set(@interface.as_json(:allow_nan => true), scope: @scope)
       if !packet.identified?
         # Need to identify so we can find the target
-        identified_packet = System.commands.identify(packet.buffer(false), @cmd_target_names)
+        identified_packet = System.commands.identify(packet.buffer(false), @interface.cmd_target_names)
```

Comment:
I guess all packets are identified because otherwise this would have crashed the router.

Reply:
Yes, when we are using the PREIDENTIFIED protocol